frontend/.env

```diff
@@ -1,4 +1,5 @@
 VITE_TITLE=GenePlexus
 VITE_DESCRIPTION=GenePlexus enables researchers to predict novel genes similar to genes of interest based on their patterns of connectivity in human genome-scale networks.
-VITE_URL=
+VITE_URL=https://geneplexus.net
```

Comment:
For what it's worth, I'm still waiting to get control over `geneplexus.net`. I don't know if you want to soft-launch on a different domain, or just optimistically change it and hope that we get control soon.

Reply:
Just wanted to put the variable in place so we can easily change it when we do the switch. No one's looking at the new site yet so should be okay to have fake values in there. If google indexes the netlify site right now it'll see a mismatch but that's okay.